source/installation/resource-manager/linuxhost.rst

```diff
@@ -1,89 +1,69 @@
+ssh
+  connects from the web node to a configured host such as a login node.
+tmux
+  Specially named ``tmux`` sessions offer the ability to rediscover running jobs
+timeout
+  is used to set a 'walltime' after which the job is killed
+pstree
+  is used to detect the job's parent ``sinit`` process so that it can be killed
+singularity
+  containerization provides a PID namespace without requiring elevated privileges that ensures that all child processes are cleaned up when the job either times out or is killed
 
-First update the PAM stack to include the following line:
+.. note:: There are many recipes for managing processes on arbitrary Linux hosts, and we will be exploring others in future development of this adapter.
 
-.. code-block:: none
 
-   session     required      pam_exec.so type=open_session /etc/security/limits.sh
+The adapter and target host can be configured to either:
 
-The following example of ``/etc/security/limits.sh`` is used by OSC on interactive login nodes. Adjust ``MemoryLimit`` and ``CPUQuota`` to meet the needs of your site. See ``man systemd.resource-control``
+#. Enable launching any software installed on the target host
+#. Enable launching a specific application container
 
-.. code-block:: bash
-
-   #!/bin/bash
-   set -e
-
-   PAM_UID=$(id -u "${PAM_USER}")
-
-   if [ "${PAM_SERVICE}" = "sshd" -a "${PAM_UID}" -ge 1000 ]; then
-           /usr/bin/systemctl set-property "user-${PAM_UID}.slice" \
-                   MemoryAccounting=true MemoryLimit=64G \
-                   CPUAccounting=true \
-                   CPUQuota=700%
-   fi
+The cluster configuration can be shared between both strategies and each interactive app plugin individually configured to use the appropriate strategy.
 
-Approach #2: libcgroup cgroups
-------------------------------
 
-The libcgroup cgroups rules and configurations are a per-group resource limit where the group is defined in the examples at ``/etc/cgconfig.d/limits.conf``. The following examples limit resources of all tmux processes launched for the LinuxHost Adapter so they all share 700 CPU shares and 64GB of RAM. This requires setting ``tmux_bin`` to a wrapper script that in this example will be ``/usr/local/bin/ondemand_tmux``.
+Launch any software on target host
+----------------------------------
 
-Example of ``/usr/local/bin/ondemand_tmux``:
+1. Install on the target host a Singularity base OS image matching the OS of the target host. In the cluster configuration example below we use ``/opt/ood/linuxhost_adapter/centos_7.6.sif`` as the target host is running CentOS7.
+2. Add the cluster configuration file for the target host
```

Comment:
Might be good to link to the cluster configuration section here. It took me a minute to realize it was further down the page when I was reading this.